test_files/delayed_import_typing.expected.py

```diff
@@ -0,0 +1,13 @@
+import typing
+
+if typing.TYPE_CHECKING:
+    import bmemcached  # noqa
```

Comment:
one problem i was having was that it was re3moving "bmemcached" for some reason, couldnt figure that out.

can we take out the "# noqa" s here?  this block should proceed unaffected

Reply:
just did it